common/eth2_network_config/built_in_network_configs/kiln/boot_enr.yaml

```diff
@@ -0,0 +1 @@
+- enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk
```

Comment:
Looks like there are two more bootnodes in the README: https://github.com/eth-clients/merge-testnets/tree/main/kiln#kiln-testnet

Reply:
Nice spotting, added in c45e6291a.